sale_stock_available_info_popup/models/sale_order.py

```diff
@@ -16,15 +16,13 @@ class SaleOrderLine(models.Model):
     @api.depends("product_id", "product_uom_qty")
     def _compute_immediately_usable_qty_today(self):
         qty_processed_per_product = defaultdict(lambda: 0)
-        remaining = self.env["sale.order.line"]
+        self.immediately_usable_qty_today = False
```

Comment:
False or 0?

Reply:
Works either way. Odoo sets similar fields to `False` by default so I stuck with that choice